main.py

```diff
@@ -1,8 +1,21 @@
 import click
 
-@click.command()
-def hello():
-    click.echo('Hello World!')
+taskcounter = 0
 
-if __name__ == '__main__':
-    hello()
+#Create a new task
+def newtask(): 
+    input("Task: ")
+    taskcounter = taskcounter +1  #I want to add +1 to the task counter but can´t
```

Comment:
Ahh. Let me explain. Here you are creating a new local variable `taskcounter`, you are not referencing the global variable `taskcounter`.  That is why python will give you an error:

`Using variable 'taskcounter' before assignment`.

Python thinks that you are referencing the new local variable `taskcounter`.

If you just want to sum 1 to your global variable you can do:

```suggestion
    taskcounter = +1
```